MacrobenchmarkSample/app/src/main/java/com/example/macrobenchmark/target/activity/MainActivity.kt

```diff
@@ -115,4 +124,22 @@ class MainActivity : ComponentActivity() {
         }
         startActivity(intent)
     }
+
+    /**
+     * Logs the app's Baseline Profile Compilation Status using [ProfileVerifier].
+     */
+    fun logCompilationStatus() {
+        CoroutineScope(Dispatchers.IO).launch {
```

Comment:
This will make this coroutine unconstrained of any lifecycle.
I'd suggest:
1. making the function `suspend`
2. use `withContext(Dispatchers.IO)`
3. in the `onCreate`, use `lifecycleScope.launch { logCompilationStatus() }`